pages/[slug].js

```diff
@@ -17,6 +18,9 @@ export default function SpotDetails() {
 
   return (
     <>
+      <Head>
+        <title>SpotiFinder: {currentSpot.name}</title>
+      </Head>
       <StyledImage>
         <Image
           src={currentSpot.image}
```

Comment:
there is an error in the console. You can fix it by adding "priority" to the Image.

Reply:
nice, thanks :)